app/assets/stylesheets/companion_window.css

```diff
@@ -209,7 +210,6 @@
     border-bottom: 1px solid gray;
 
     background-color: var(--background-color);
-    padding-right: 4px;
```

Comment:
Why was this padding removed?

Reply:
It made the drop-shadow look weird. I moved the padding to the nav so we don't have that problem. 
![Screenshot 2025-02-10 at 4 36 31 PM](https://github.com/user-attachments/assets/28a88f8c-714b-4945-8f99-b809d869ff79)